src/components/styled/StyledDoneLabel.js

```diff
@@ -4,3 +4,5 @@ export const StyledDoneLabel = styled.label`
 	flex: 1;
 	text-decoration: ${props => (props.done ? `line-through` : `none`)};
 `;
+
+export const StyledInputLabel = styled.label``;
```

Comment:
this is empty, so you dont need to make it a styled component

Reply:
Thanks, I removed the unnecessary component